exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/MockTraceServiceClient.java

```diff
@@ -0,0 +1,33 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.devtools.cloudtrace.v2.BatchWriteSpansRequest;
+import com.google.devtools.cloudtrace.v2.ProjectName;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TraceServiceGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+
+import java.util.List;
+
+public class MockTraceServiceClient {
```

Comment:
Unlike Python, `TraceServiceClient` was very difficult to work with, as it has a lot of setup involving TLS/SSL, GCP credentials, etc. and I was unable to get it to connect to the mock server. So, I created this simplified version of it to be used for tests

Reply:
That's really unfortunate, would be nice to find a way to use the standard client with mock server.
Fine to work on that after this PR is merged though.